easybuild/easyconfigs/n/NAGfor/NAGfor-6.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Binary'
+
+name = 'NAGfor'
+version = '6.2'
+
+homepage = 'http://www.nag.co.uk'
+description = """The checking compiler for improved code portability and detailed error reporting."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+nagname = 'npl%(version_major)sa%(version_major)s%(version_minor)sna_amd64'
+
+source_urls = ['https://www.nag.co.uk/downloads/impl']
+sources = [nagname + '.tgz']
+checksums = ['4fce371e799d321c7842f867b8a98e64d54cc806da43d801c77e3645f295c1d4']
+
+preinstallopts = 'tar -xvf %s.tgz && ' % nagname
```

Comment:
@SimonPinches Same here as in the `NAG` easyconfig, use `PackedBinary` to simplify this a bit and avoid the use of the local `nagname` variable?

Reply:
Again, I failed to get the installation to work using `<PackedBinary>`.